util/fqdn.go

```diff
@@ -7,18 +7,23 @@ import (
 	"strings"
 )
 
-func removeQuestionMarks(domain string) string {
+func RemovePrependedQuestionMarks(domain string) string {
 	for strings.HasPrefix(domain, "?.") {
 		domain = domain[2:]
 	}
 	return domain
 }
 
-func IsFQDN(domain string) bool {
-	domain = removeQuestionMarks(domain)
+func RemoveWildcardFromDomain(domain string) string {
```

Comment:
Can you name these two functions consistently?

Reply:
done